api/api.yaml

```diff
@@ -2835,6 +2848,75 @@ components:
           description: The time the connection was last updated.
           format: date-time
           example: 2023-07-13T21:34:44.816354Z
+        authScheme:
+          type: string
+          description: The authentication scheme used for this connection.
+          example: oauth2
+          enum:
+            [
+              none,
+              apiKey,
+              basic,
+              oauth2/authorizationCode,
+              oauth2/authorizationCodePKCE,
+              oauth2/clientCredentials,
+              oauth2/password,
+              jwt,
+            ]
+        scopes:
```

Comment:
let's actually remove this, it should only show up nested within fields like `OAuth2AuthorizationCode` since not applicable for all Connections.